app/import.go

```diff
@@ -49,7 +49,8 @@ func processAttachmentPaths(files *[]AttachmentImportData, basePath string, file
 			*f.Path = path
 			if len(filesMap) > 0 {
 				if (*files)[i].Data, ok = filesMap[path]; !ok {
-					return fmt.Errorf("attachment %q not found in map", path)
+					mlog.Warn(fmt.Sprintf("attachment %v not found and will not be imported", path))
```

Comment:
```suggestion
					mlog.Warn(fmt.Sprintf("attachment %s not found and will not be imported", path))
```
Do this change everywhere.